code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -75,16 +75,15 @@ GLOBAL_LIST_EMPTY(antagonists)
 	set_antag_hud(mob_override, null)
 
 // Handles adding and removing the clumsy mutation from clown antags. Gets called in apply/remove_innate_effects
-/datum/antagonist/proc/handle_clown_mutation(mob/living/mob_override, message)
+/datum/antagonist/proc/handle_clown_mutation(mob/living/mob_override, message, removing = TRUE)
 	var/mob/living/carbon/human/H = mob_override
-	if(H.mind.assigned_role == "Clown")
-		if(H && istype(H))
-			if(H.dna.check_mutation(CLOWNMUT)) // They're a clown becoming an antag, remove clumsy
-				if(!silent && message)
-					to_chat(H, "<span class='boldnotice'>[message]</span>")
-				H.dna.remove_mutation(CLOWNMUT)
-			else
-				H.dna.add_mutation(CLOWNMUT) // We're removing their antag status, add back clumsy
+	if(H && H.mind.assigned_role == "Clown")
+		if(removing) // They're a clown becoming an antag, remove clumsy
```

Comment:
You should still check if they are actually human

Reply:
oops, you're right, putting that check back